risc0/circuit/recursion/src/control_id.rs

```diff
@@ -14,7 +14,7 @@
 
 /// Merkle root of the RECURSION_CONTROL_IDS
 pub const ALLOWED_IDS_ROOT: &str =
-    "6df708447638d36828ebf4545980ff39315562181c926d3a9e2697405f3acf15";
+    "576e1600c81b7c1c0b50e63550420019a5f92445c0ca180c9de4d5723f8c8020";
```

Comment:
I might be missing something here but why was this changed? I thought this PR only changed the rv32im

Reply:
Good catch. I actually found a bug in the loader that was producing the control group. The "bug" didn't effect anything to do with proving, so the impact was low risk. But by fixing it this causes all the control IDs to be updated.